tracim/tracim/controllers/admin/user.py

```diff
@@ -326,7 +328,7 @@ def post(
             user.password = password
         elif send_email:
             # Setup a random password to send email at user
-            password = str(uuid.uuid4())
+            password = self.generate_password()
```

Comment:
better to use parameters here than in the method itself.

if you define generate_password(lenght, allowed_chars), then the method may be used in another context. If all stuff is hard written then you can't. See comment below